apps/docs/content/guides/platform/read-replicas.mdx

```diff
@@ -1,99 +1,76 @@
 ---
 title: 'Read Replicas'
-description: 'Spin up additional read-only databases for your project.'
-subtitle: 'Spin up additional read-only databases for your project'
+description: 'Deploy additional read-only databases in multiple regions for your project.'
+subtitle: 'Deploy additional read-only databases in multiple regions for your project'
 ---
 
-<Admonition type="caution" label="Read replicas are only currently available for Early Access for projects on the Pro Plan and above.">
+<Admonition type="caution" label="Read Replicas are currently available to organizations on Pro plan and above who have signed up for early access.">
```

Comment:
```suggestion
<Admonition type="note" label="Read Replicas are available to organizations on Pro plan and above who have signed up for early access.">
```

according to our style guide, `caution` is for things that can cause serious user inconvenience (but short of data loss or leak), I'd say this is definitely a `note`